recipes/liquid-dsp/all/conanfile.py

```diff
@@ -0,0 +1,170 @@
+    name = "liquid-dsp"
+    description = "Digital signal processing library for software-defined radios "
+    topics = ("conan", "dsp", "sdr", "liquid-dsp")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jgaeddert/liquid-dsp"
+    license = ("MIT",)
+    settings = "os", "arch", "build_type", "compiler"
+    exports_sources = ["generate_link_library.bat", "patches/**"]
+    options = {
+        "shared": [True, False],
+        "simdoverride": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "simdoverride": False,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _libname(self):
+        if self.settings.os == "Windows":
+            return "libliquid"
+        return "liquid"
+
+    @property
+    def _target_name(self):
+        if self.settings.os == "Macos":
+            if not self.options.shared:
+                return "libliquid.ar"
+            return "libliquid.dylib"
+        if not self.options.shared:
+            return "libliquid.a"
+        return "libliquid.so"
+
+    @property
+    def _lib_pattern(self):
+        if self.settings.os == "Macos" and not self.options.shared:
+            return "libliquid.a"
+        if self.settings.os != "Windows":
+            return self._target_name
+        return "libliquid.lib"
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("mingw-w64/8.1")
+            self.build_requires("automake/1.16.3")
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            destination=self._source_subfolder,
+            strip_root=True,
+        )
+
+    def _patch_sources(self):
+        if self.settings.os == "Windows":
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
+
+    def _gen_link_library(self):
+        if self.settings.compiler != "Visual Studio" or (not self.options.shared):
+            return
+        self.run("cmd /c generate_link_library.bat")
+        with tools.chdir(self._source_subfolder):
+            self.run(
+                "{} /def:libliquid.def /out:libliquid.lib /machine:{}".format(
+                    os.getenv("AR"), "X86" if self.settings.arch == "x86" else "X64"
+                ),
+                win_bash=tools.os_info.is_windows,
+            )
+
+    def _rename_libraries(self):
+        with tools.chdir(self._source_subfolder):
+            if self.settings.os == "Windows" and self.options.shared:
+                os.rename("libliquid.so", "libliquid.dll")
```

Comment:
```suggestion
                tools.rename("libliquid.so", "libliquid.dll")
```

https://docs.conan.io/en/latest/reference/tools.html#tools-rename